plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -104,16 +99,12 @@ func defaultPredicates() sets.String {
 		factory.RegisterFitPredicateFactory(
 			"NoVolumeZoneConflict",
 			func(args factory.PluginFactoryArgs) algorithm.FitPredicate {
-				return predicates.NewVolumeZonePredicate(args.NodeInfo, args.PVInfo, args.PVCInfo)
+				return predicates.NewVolumeZonePredicate(args.PVInfo, args.PVCInfo)
 			},
 		),
 		// Fit is determined by node selector query.
-		factory.RegisterFitPredicateFactory(
-			"MatchNodeSelector",
-			func(args factory.PluginFactoryArgs) algorithm.FitPredicate {
-				return predicates.NewSelectorMatchPredicate(args.NodeInfo)
-			},
-		),
+		// TODO: This is run as part of GeneralPredicates - figure out the way to remove it.
```

Comment:
I think moving this to `init()` is safe enough for backward compatibility. I should have moved it in #20204, but I forgot to do it, sorry.